src/app/client/src/app/modules/observation/service/observation-util/observation-util.service.spec.ts

```diff
@@ -0,0 +1,12 @@
+import { TestBed } from '@angular/core/testing';
+
+import { ObservationUtilService } from './observation-util.service';
+
+xdescribe('ObservationUtilService', () => {
```

Comment:
Why this test block is skipped. Please fix this.

Reply:
we haven't implemented the tets cases for 4.0 release